bonus_guides/F_config.md

```diff
@@ -1 +1,46 @@
-## FPO
+Since our Phoenix apps are simply Elixir applications, they  have the same configuration and structure as other Mix projects. Recall that Mix is the build tool used by most Exixir Apps.
+
+The config for our Phoenix apps is stored in a combination of .ex and .exs files.
+
+ `.ex` and  `.beam` Files
+
+Our `.ex` files are stored in the `/lib` folder of our App. These files are used to store high level configuration for our apps, such as endpint and OTP supervisor/worker config. 
+
+The `.ex` files are Elixir scripts, which are be compiled to `.beam` files that run on the Erlang Virtual Machine (BEAM, aka Bodgan's Erlang Abstract Machine). Once compiled these `.beam` files will be located in the `_build` folder of our App, the `.beam` files will be located in the `/ebin` subfolder for each Mix project in our app. For instance, since Phoenix itself is just another Elixir app, the `.beam` files for its many Mix projects are located in `/_build/dev/lib/phoenix/ebin`. Note that `/dev` would be interchanged with `/test` or `/prod` for our different environments. Also note that `phoenix` would be interchanged with the name of our app, which is also just going to be another Elixir app.
+
+`.exs` Files
+
+Our `.exs` files are stored in the `/` root folder and the `/config` folder. These files are used to store the primary configuration details for our app and each of our environments.
+
+The `.exs` files are Elixir scripts, which are not compiled to `.beam` files. Instead, the compilation of `.exs` files happens in memory each time they are run, which is makes them useful for scripts that change often or or standalone tasks during development.
+
+Our `mix.exs` file is located in the root folder of our app and contains important config details relating to our app, including compilation paths, dependencies and aliases.
+
+#### Environments
+
+Phoenix applications are intended to be run with configuration particular to different environments, such as 'development' or 'production'. This allows for an improved developer experience and smoother workflow when deploying apps to production. By default, the `/config` folder at the root of our apps our new apps will include the following configuration files:
+
+`config.exs`, which is our master config file that contains logging and endpoint details, such as our app url and root directory.
```

Comment:
What would you think about mentioning the idea that this file is for configuration which is common across all environments. There's an import call at the bottom which brings in configuration specific to the current environment?

I'm referring to this part:
# Import environment specific config. This must remain at the bottom
# of this file so it overrides the configuration defined above.
import_config "#{Mix.env}.exs"